chain/network/src/peer.rs

```diff
@@ -263,47 +265,54 @@ impl Peer {
             PeerMessage::BlockHeaders(headers) => {
```

Comment:
Why is block header not routed?

Reply:
Discussed with @SkidanovAlex . It is not needed for Blocks/BlockHeaders, you should be able to get them from neighboring peers, no need to request them from nodes farther.